functions/Import-DbaCsv.ps1

```diff
@@ -124,12 +124,12 @@ function Import-DbaCsv {
         The progress bar is pretty but can slow down imports. Use this parameter to quietly import.
 
     .PARAMETER Quote
-        Defines the default quote character wrapping every field.
+        Defines the default quote character wrapping every field.  Default is ".
 
     .PARAMETER Escape
         Defines the default escape character letting insert quotation characters inside a quoted field.
 
-        The escape character can be the same as the quote character.
+        The escape character can be the same as the quote character.  Default is ".
```

Comment:
These seem to be worded a bit wrong, but best to use the word instead of the character to make it clear. Some editors will look like single quotes based on fonts.

I think rewording the whole thing will make it a bit clearer:

```
.PARAMETER Escape
    Define the escape character allowing quotation characters inside a quoted field. 
    Default: double-quotes. 
    
    The escape character can be the same as the quote character.
```

Reply:
Could do the same to the Comment parameter help

```
.PARAMETER Comment
    Define the comment character indicating that a line is commented out.
    Default: hashtag
```